### PR TITLE
Feature: Updated docs. Added missing methods in HandlerBase.

### PR DESCRIPTION
**Pull request by MartynasZilinskas**
* Updated `/docs/`
* Added missing methods in `HandlerBase`
* Removed empty interfaces in Section and Header component bases

### CODE REVIEW

File: src/abstractions/handler-base.ts [hunk 1]
```diff
@@ -212,4 +212,43 @@ export abstract class HandlerBase<
 
         return restProps;
     }
+
+    //#region Public API
+    /**
+     * Get a boolean if dropdown is open or not.
```
Comment (DeividasBakanas) [inline src/abstractions/handler-base.ts]: Maybe: 
`Get a boolean whether dropdown is open or not.`

File: src/abstractions/handler-base.ts [hunk 2]
```diff
@@ -212,4 +212,43 @@ export abstract class HandlerBase<
...
+    //#region Public API
+    /**
+     * Get a boolean if dropdown is open or not.
+     */
+    public isOpen(): boolean {
+        return this.state.isOpen;
+    }
+
+    /**
+     * To close dropdown.
```
Comment (DeividasBakanas) [inline src/abstractions/handler-base.ts]: `To open dropdown`.

File: src/abstractions/handler-base.ts [hunk 3]
```diff
@@ -212,4 +212,43 @@ export abstract class HandlerBase<
...
+    public close(): void {
+        if (!this.state.isOpen) {
+            return;
+        }
+
+        this.setState((state: HandlerBaseState) => ({
+            ...state,
+            isOpen: false
+        }));
+        this.triggerCallbacks(true, EventSource.ManualTrigger);
```
Comment (DeividasBakanas) [inline src/abstractions/handler-base.ts]: False:

```tsx
this.triggerCallbacks(false, EventSource.ManualTrigger);
```

File: src/abstractions/handler-base.ts (hunk 1 above)
Comment (MartynasZilinskas) [reply src/abstractions/handler-base.ts]: `Is dropdown open.`

**Pull request merged**